perfkitbenchmarker/linux_packages/cassandra.py

```diff
@@ -103,6 +104,8 @@ def Configure(vm, seed_vms):
              'num_cpus': vm.num_cpus,
              'cluster_name': 'Test cluster'}
 
+  vm.UpdateVMHostsFile()
```

Comment:
As far as I remember I've been receiving
```
Error: Exception thrown by the agent : java.net.MalformedURLException: Local host name unknown: java.net.UnknownHostException
```
I'm not able to test it at the moment, I'll try ASAP.